### PR TITLE
Release 0.8

### PR DESCRIPTION
**Pull request by CPerezz**
Resolves: #128

### CODE REVIEW

File: .github/workflows/dusk_ci.yml [hunk 1]
```diff
@@ -36,7 +36,7 @@ jobs:
       - uses: actions-rs/cargo@v1
         with:
           command: test
-          args: --release
+          args: --release
```
Comment (ZER0) [inline .github/workflows/dusk_ci.yml:39]: remove the trailing space.

File: .github/workflows/dusk_ci.yml [hunk 2]
```diff
@@ -53,7 +53,7 @@ jobs:
       - uses: actions-rs/cargo@v1
         with:
           command: test
-          args: --release --no-default-features
+          args: --release --no-default-features
```
Comment (ZER0) [inline .github/workflows/dusk_ci.yml:56]: remove the trailing space.

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,21 +7,38 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+- Change error module to include plonk & poseidon err variants [#131](https://github.com/dusk-network/dusk-blindbid/issues/131)
```
Comment (ZER0) [inline CHANGELOG.md:14]: Use one approach, either you put all the links at the end or all the links in the same line.
This is comment is valid for the entire changelog.

File: CHANGELOG.md [hunk 4]
```diff
@@ -7,21 +7,42 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.8.0] - 07-06-21
```
Comment (ZER0) [inline CHANGELOG.md:10]: The date should be in the ISO 8601 format.

File: Cargo.toml [hunk 5]
```diff
@@ -18,43 +18,34 @@ exclude = [
...
-canonical_derive = { version = "0.5", optional = true }
-anyhow = {version = "1", optional = true}
+dusk-pki = { version = "0.7.0-rc", default-features = false }
+dusk-poseidon = { version = "0.21.0-rc", default-features = false }
+dusk-bls12_381 = { version = "0.8", default-features = false }
+dusk-jubjub = { version = "0.10", default-features = false }
+num-bigint = { version = "0.3", optional = true } 
+num-traits = { version = "0.2", optional = true } 
+rand_core = { version = "0.6", default-features = false }  
+lazy_static = "1.4"
```
Comment (ZER0) [inline Cargo.toml:28]: Removed `lazy_static`

**Review by ZER0 — APPROVED** [review]
LGTM with the nits addressed

**Comment by CPerezz** [comment]
Blocked until https://github.com/dusk-network/dusk-bytes/pull/21 is merged and a new patch is released.

**Pull request merged**